src/content/docs/blog/tauri-board-elections-2024.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Tauri Board Elections 2024
+date: 2024-06-28
+authors: [jbolda]
+excerpt: The Tauri Programme is celebrating it's third anniversary of Tauri becoming a programme within The Commons Conservancy. We are hard at work bringing v2 to a stable release, and now the next round of Tauri Board Director elections is upon us!
+banner:
+  content: |
+    You're reading the blog on the prerelease site for Tauri 2.0 -
+    <a href="https://tauri.app">Go to the Tauri 1.0 site</a>
```

Comment:
Not sure if/how replicating to the V1 site is needed.

Reply:
v1 points to the v2 docs, so no need.